packages/cli/templates/webcomponents/igc-ts/projects/_base/files/package.json

```diff
@@ -21,9 +21,9 @@
   },
   "dependencies": {
     "@vaadin/router": "^1.7.4",
-    "lit": "^3.1.2",
-    "typescript": "^5.3.3",
-    "igniteui-webcomponents": "~4.11.0"
+    "lit": "^3.2.0",
```

Comment:
For WC I get 
![image](https://github.com/user-attachments/assets/731e5035-baaa-48bb-bc59-39c652d3b4f3)

This is because the `igc-form` component has been removed in https://github.com/IgniteUI/igniteui-webcomponents/pull/1302 so we need to use the native form in our templates.